pkg/cmd/inletsoperator_app.go

```diff
@@ -30,12 +41,60 @@ func makeInstallInletsOperator() *cobra.Command {
 		namespace, _ := command.Flags().GetString("namespace")
 
 		if namespace != "default" {
-			return fmt.Errorf(`to override the namespace, edit the YAML files on GitHub`)
+			return fmt.Errorf(`to override the namespace, install inlets-operator via helm manually`)
 		}
+
+		arch := getArchitecture()
+		fmt.Printf("Node architecture: %q\n", arch)
+
+		userPath, err := config.InitUserDir()
+		if err != nil {
+			return err
+		}
+
+		clientArch, clientOS := getClientArch()
+
+		fmt.Printf("Client: %q, %q\n", clientArch, clientOS)
+
+		log.Printf("User dir established as: %s\n", userPath)
+
+		os.Setenv("HELM_HOME", path.Join(userPath, ".helm"))
+
+		_, err = tryDownloadHelm(userPath, clientArch, clientOS)
+		if err != nil {
+			return err
+		}
+
+		err = addHelmRepo("inlets", "https://inlets.github.io/inlets-operator/")
+		if err != nil {
+			return err
+		}
+
+		updateRepo, _ := inletsOperator.Flags().GetBool("upadte-repo")
+
+		if updateRepo {
+			err = updateHelmRepos()
+			if err != nil {
+				return err
+			}
+		}
+
+		chartPath := path.Join(os.TempDir(), "charts")
+
+		err = fetchChart(chartPath, "inlets/inlets-operator")
```

Comment:
add a flag for this conditionally like cert-manager and openfaas apps are doing

Reply:
Done!